test_scripts/API/VehicleData/GetVehicleData/008_GetVD_app_version_is_less_than_parameter_version.lua

```diff
@@ -0,0 +1,44 @@
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL rejects vehicle data RPCs with <vd_param> parameter
+-- if an app is registered with version less than current parameter version
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) Vehicle Data RPCs and <vd_param> parameter are allowed by policies
+-- 3) <vd_param> parameter has param_version = 5.1.0
```

Comment:
https://github.com/dboltovskyi/sdl_atf_test_scripts/pull/13/files#r454921572 the same comment here